utils/gyb_sourcekit_support/UIDs.py

```diff
@@ -272,6 +275,8 @@ def __init__(self, internal_name, external_name):
     REQUEST('Diagnostics', 'source.request.diagnostics'),
     REQUEST('Compile', 'source.request.compile'),
     REQUEST('CompileClose', 'source.request.compile.close'),
+    REQUEST('ExpandMacroSyntactically',
+            'source.request.expand.macro.syntactically'),
```

Comment:
We are... quite inconsistent with these. How about `source.request.syntactic_macro_expansion`?